### PR TITLE
fix: update ngx-formly

### PR DESCRIPTION
**Pull request by PascalRepond**
- Closes https://github.com/rero/rero-ils/issues/3712
- Closes https://github.com/rero/rero-ils/issues/3705

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,8 +24,8 @@
     "@angular/platform-browser-dynamic": "^17.1.0",
     "@angular/router": "^17.1.0",
     "@biesbjerg/ngx-translate-extract-marker": "^1.0.0",
-    "@ngx-formly/core": "6.3.2",
-    "@ngx-formly/primeng": "6.3.2",
+    "@ngx-formly/core": "6.3.6",
+    "@ngx-formly/primeng": "6.3.6",
```
Comment (Garfield-fr) [inline package.json:28]: I will reset the constraint with `^6.3.2`.

File: package.json [hunk 2]
```diff
@@ -57,7 +57,7 @@
     "@angular/cli": "^17.1.0",
     "@angular/compiler-cli": "^17.1.0",
     "@angular/language-service": "^17.1.0",
-    "@ngx-formly/schematics": "6.3.2",
+    "@ngx-formly/schematics": "6.3.6",
```
Comment (Garfield-fr) [inline package.json:60]: I will reset the constraint with `^6.3.2`.

File: projects/rero/ng-core/package.json [hunk 3]
```diff
@@ -20,8 +20,8 @@
     "@angular/common": "^17.1.0",
     "@angular/core": "^17.1.0",
     "@biesbjerg/ngx-translate-extract-marker": "^1.0.0",
-    "@ngx-formly/primeng": "6.3.2",
-    "@ngx-formly/core": "6.3.2",
+    "@ngx-formly/primeng": "6.3.6",
+    "@ngx-formly/core": "6.3.6",
```
Comment (Garfield-fr) [inline projects/rero/ng-core/package.json:24]: I will reset the constraint with `^6.3.2`.

**Review by Garfield-fr — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (PascalRepond) [reply package.json:28]: `6.3.2` to `6.3.5` have bugs though. Better `^6.3.6`?

Comment (Garfield-fr) [reply package.json:28]: With this solution, he will always take the last one. You can also set the constraint as you mentioned.

**Pull request merged**